pure_pursuit_controller:
  ros__parameters:
    lookahead_distance: 0.2
    linear_velocity: 0.3
    max_angular_velocity: 1.2
    goal_tolerance: 0.15
    min_lookahead_distance: 0.1
    max_lookahead_distance: 0.3
    steering_gain: 1.5
